src/main/java/org/alfresco/rest/requests/Node.java

```diff
@@ -590,6 +590,18 @@ public RestRenditionInfoModelCollection getNodeVersionRenditionsInfo(String vers
     }
 
 
+    /**
+     * Delete the rendition identified by renditionId using DELETE call on '/nodes/{nodeId}/renditions/{renditionId}
```

Comment:
Close the quotes on the URI within the Javadoc.

Reply:
Done.